wp-content/themes/access/src/js/field.js

```diff
@@ -44,6 +45,11 @@ import Toggle from 'utilities/toggle/toggle';
   $(ResultsField.Selectors.DOM).each((i, el) =>
     new ResultsField(el).init());
 
+  // Initialize tooltips.
+  $(`.${Tooltip.CssClass.TRIGGER}`).each((i, el) =>
+    new Tooltip(el).init());
+
+
   /** Initialize ACCESS NYC Patterns library components */
   new Icons('/wp-content/themes/access/assets/svg/icons.475e6e65.svg');
   new Accordion();
```

Comment:
@theasteve this doesn't look consistent with the TEST branch. The Tooltips should not be included in this script.